KsApi/PKCE.swift

```diff
@@ -0,0 +1,58 @@
+import CryptoKit
+import Foundation
+
+/// PKCE stands for Proof Key for Code Exchange.
+/// The code verifier, and its associated challenge, are used to ensure that an oAuth request is valid.
+/// See this documentation for more details: https://www.oauth.com/oauth2-servers/mobile-and-native-apps/authorization/
+public struct PKCE {
```

Comment:
This doesn't need to be a struct; I'm just doing it for the sake of namespace clarity. These methods are called like `PKCE.createCodeVerifier(ofLength:)`.